### ISSUE TITLE
My general comments on docs and metadata

### ISSUE THREAD

**Issue by mulkieran** [stratis-storage/stratis-docs#15]
* I wish MDAA and MDAB were something like MDAl and MDAr (for left and right).
* I wish I knew where the Stratis signauture came from.
* I think the length of a sector should be defined in the text, where it is used to specify the length of the areas. (It is defiend in the table).
* I think purpose of unused should be explained.
* I do not know why flags and unused are grouped in the item that says both are zeroed, especially as that is their only mention. Flags for what?
* You're a bit inconsistant in naming.
  * CRC should probably be CRC32 everywhere.
  * Sometimes you use MDA to stand for one half of the MDA, and sometimes you use it to stand for the whole thing.
* A diagram of the arrangement of metadata on the blockdev would be nice, but maybe better for later.
* What is reserved space reserved for?
* Keep the offsets in the relevant tables! They are helpful.
* It might be nice to group the MDAB and MDAA items in the Signautre block table visualyl, perhaps with a double line.

**Comment by agrover** [comment]
Hi, thanks for your detailed comments I think I addressed everything, but had a few responses, below.

> I wish I knew where the Stratis signauture came from.

I made it up. I think the requirement is just that it be an sequence unlikely to occur elsewhere, since that would be very bad. (and I included a mangled form of "stratis" to make it more apparent in hex dumps.)

> Keep the offsets in the relevant tables! They are helpful.

I think they are, and you're saying "good job"? Or are they missing somewhere?
